idl/transaction.x

```diff
@@ -19,6 +19,9 @@ namespace mazzaroth
   {
     // Contract binary bytes.
     opaque contract<>;
+
+    // Schema data format for use with keyquery data formats
+    Schema schema;
```

Comment:
Do you not have to include the schema.x file to use the objects from there?

Was this changed at some point?

Reply:
The include statements don't actually do anything. Instead we just have to select all the .x files we want included.

Reply:
Oh okay did not know that.

Reply:
We could probably enforce it at some point, but for now it's basically just a comment.